crates/models/llama/src/lib.rs

```diff
@@ -185,7 +178,7 @@ impl KnownModel for Llama {
                     n_rot,
                     0,
                 );
-                ggml::set_name(&q_current, "Qcur");
+                ggml::set_tensor_name(&q_current, "Qcur");
```

Comment:
ah I see, these calls were there before. We should make `set_tensor_name` private to `ggml`, and force everyone to use `Tensor::set_name`.

Reply:
Should we change this in this PR or create another one?

Reply:
I'll create another one after this to fix up the names. Don't worry about it.